java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -348,6 +348,23 @@ public final ColumnVector isNull() {
     return new ColumnVector(isNullNative(getNativeView()));
   }
 
+  /**
+   * Returns a Boolean vector with the same number of rows as this instance, that has
```

Comment:
nit: can we clarify this a bit more? It is not clear what the input type should be or really what this does in terms of checks. The following from the C++ docs I think explains it better, but could possibly be paired down a bit too. Also it looks like it still is explaining the old behavior. Wasn't it updated to work with exponent format? like '+17E30'?

```
 * @brief Returns a boolean column identifying strings in which all
 * characters are valid for conversion to fixed-point.
 *
 * The output row entry is set to `true` if the corresponding string element
 * has at least one character in [+-0123456789.]. The optional sign character
 * must only be in the first position. The decimal point may only appear once.
 * Also, the integer component must fit within the size limits of the
 * underlying fixed-point storage type. The value of the integer component
 * is based on the scale of the `decimal_type` provided.
 *
 * @code{.pseudo}
 * Example:
 * s = ['123', '-456', '', '1.2.3', '+17E30', '12.34' '.789', '-0.005]
 * b = is_fixed_point(s)
 * b is [true, true, false, false, false, true, true, true]
 * @endcode
 *
 * Any null entries result in corresponding null entries in the output column.
```

Reply:
I have updated both the docs. PTAL before I merge